### PR TITLE
Bedriye

### PR DESCRIPTION
**Pull request by bedriye1**
Hello!

### CODE REVIEW

**Comment by mstklcrsln** [comment]
very good let's merge it

File: src/test/java/com/fleetGru/Pages/FleetVehicles.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ public FleetVehicles() {
     @FindBy(xpath = "(//*[@class='fa-refresh'])[2]")
     public WebElement resetBtn;
 
-    @FindBy(xpath = "//td[.='Samuel Christiansen']")
-    public WebElement SamuelChrist;
+    @FindBy(xpath = "//tr[@class='grid-row row-click-action']")
```
Comment (Mustafa80baz) [inline src/test/java/com/fleetGru/Pages/FleetVehicles.java]: It looks okay

File: src/test/java/com/fleetGru/Pages/FleetVehicles.java [hunk 2]
```diff
@@ -61,7 +61,7 @@ public FleetVehicles() {
...
+    public WebElement firstRow;
```
Comment (Mustafa80baz) [inline src/test/java/com/fleetGru/Pages/FleetVehicles.java]: it is okay

File: src/test/java/com/fleetGru/StepDefinitions/AddEventStepDefinitions.java [hunk 3]
```diff
@@ -19,6 +19,7 @@ public class AddEventStepDefinitions extends BasePage {
     @When("User can go to fleet dropdown")
     public void userCanGoToFleetDropdown() {
         waitUntilLoaderScreenDisappear();
+        BrowserUtils.sleep(5);
```
Comment (Mustafa80baz) [inline src/test/java/com/fleetGru/StepDefinitions/AddEventStepDefinitions.java:22]: AS

File: src/test/java/com/fleetGru/StepDefinitions/AddEventStepDefinitions.java [hunk 4]
```diff
@@ -32,13 +33,13 @@ public void userClickTheVehiclesModules() {
     @When("User can click on any row in the table")
     public void user_can_click_on_any_row_in_the_table() {
        BrowserUtils.sleep(5);
-       fleetVehicles.SamuelChrist.click();
+       fleetVehicles.firstRow.click();
```
Comment (Mustafa80baz) [inline src/test/java/com/fleetGru/StepDefinitions/AddEventStepDefinitions.java:36]: okay

File: src/test/java/com/fleetGru/StepDefinitions/AddEventStepDefinitions.java [hunk 5]
```diff
@@ -32,13 +33,13 @@ public void userClickTheVehiclesModules() {
...
     }
 
     @Then("User click the add event button")
     public void user_click_the_add_event_button() {
       BrowserUtils.sleep(5);
-      WebElement addEvent = Driver.getDriver().findElement(By.xpath("(//div[@class='pull-left btn-group icons-holder'])[3]"));
+      WebElement addEvent = Driver.getDriver().findElement(By.xpath("//a[@title='Add an event to this record']"));
```
Comment (Mustafa80baz) [inline src/test/java/com/fleetGru/StepDefinitions/AddEventStepDefinitions.java:42]: okay

**Review by Mustafa80baz — APPROVED** [review]

**Pull request merged**